drafts/build_a_dao_on_fuel/draft.md

```diff
@@ -0,0 +1,1459 @@
+
+# How to Build a DAO on the Fuel Blockchain.
+
+## Introduction
+
+A Decentralized Autonomous Organization (DAO) is considered an essential blockchain application.
+
+It is a Decentralized application that users or organizations use to automate governance execution of their projects and communities
+It is an automated program represented by rules encoded in a computer that is transparently controlled by the organization members and not influenced by any central authority.
+
+On completing this article, you will learn how to build a full-stack DAO on the fuel blockchain; you will touch down on the following concepts of development;
+
+1. Writing a DAO smart contract using the sway language.
+2. Wallet configuration
+3. Compiling and deploying to the fuel blockchain (testnet).
+4. Creating your DAO front end, including contract calls from your deployed contract. 
+
+## Prerequisites
+
+To follow through this tutorial seamlessly, you must have the following checked out.
+
+1. [Installation](https://docs.fuel.network/guides/installation/) of dependencies like `fuel`, `sway` extensions on vscode, `rustc`, `cargo`, `rustup` `cargo-generate`, on your pc.
+2. Node version ^18.18.2 || ^20.0.0.
+3. You should have `wsl` installed if you are using a Windows or Mac PC; you can use the code `wsl --install` to download `wsl`.
+4. Knowledge of fuelVM and other fuel tools.
+5. Basic concept of blockchain technology.
+6. Writing, compiling, and deploying to any blockchain (EVM or non-EVM compatible blockchain).
+7. Understand concepts of ABIs, Faucets, wallet connect, etc.
+
+## Getting Started
+
+Before you delve into the juicy part, here are some concepts to jug your memory on the fuel ecosystem and help you better understand words you will see throughout this tutorial.
+
+**Fuel**: [Fuel](https://fuel.network/) is an operating system purpose-built for Ethereum rollups, designed to help developers build decentralized economies at scale.
+
+**FuelVM**: The [Fuel Virtual Machine](https://medium.com/blockchain-capital-blog/exploring-the-fuelvm-86cf9ccdc159) responsible for managing all code execution on the Fuel Blockchain
+
+**Sway**: [Sway](https://fuellabs.github.io/sway/) is a domain-specific language used for writing smart contracts on the fuel blockchain.
+
+**ABI**: These are called [Application Binary Interfaces](https://docs.fuel.network/guides/intro-to-sway/contract-abi/); they are the blueprint or interaction models of your smart contract, it is a layout of the detailed functionalities in your smart contract, and how to interact with it.
+
+## Setting up your Environment
+
+In the next few steps, you will set up your development environment to begin building your smart contract and the front end of your Dao.
+
+1. Go to your terminal and run the command `wsl` to start your `wsl` terminal
+
+2. Next, run the command to generate your new contract directory.
```

Comment:
The following list is not a single command, but a series of commands that must be executed sequentially. Please consider adding those clarifications.